docs/TUTORIAL.md

```diff
@@ -1,3 +1,305 @@
 # Tutorial
 
-Coming soon!
+The **ksonnet** [readme][readme] 
```

Comment:
hmm...I think of it as indicative as a reader (e.g. oh they think I should have read that? I haven't so maybe I should open it up in a tab just in case). However, this is personal preference so if you feel strongly I'm fine leaving as is.